menelaus/data_drift/psi_detector.py

```diff
@@ -0,0 +1,128 @@
+from menelaus.detector import BatchDetector
+import pandas as pd
+import numpy as np
+import copy
+
+
+class PSI_Detector(BatchDetector):
+    input_type = "batch"
+
+    def __init__(
+        self,
+    ):
+        # This initializes batch detector's parent class
```

Comment:
- [ ] Parameterize the threshold, adding it as an arg here
- [ ] Docstring for init